ui/widgets/top-bar/src/components/Topbar.tsx

```diff
@@ -53,7 +53,7 @@ const Topbar: React.FC<ITopbarProps> = props => {
     flex justify-between items-center w-full
     py-1.5 px-2 space-x-4
     border(1 grey8) rounded-none sm:rounded-md shadow-sm
-    bg-white dark:bg-black
+    bg-white dark:(bg-grey2 border(1 grey2) shadow-[0_0_4px_rgba(0,0,0,0.2)]))
```

Comment:
why are you using a custom value for the shadow here? we have an elevation prop in the twind color config, does the topbar require a different elevation from the rest of the cards?

Reply:
Hi Vali. Thanks for pointing it out. I have switched to using the BasicCardBox component instead.